src/main/java/io/confluent/connect/hdfs/TopicPartitionWriter.java

```diff
@@ -776,8 +778,7 @@ private void commitFile(String encodedPartition) {
     storage.commit(tempFile, committedFile);
     startOffsets.remove(encodedPartition);
     offsets.remove(encodedPartition);
-    offset = offset + recordCounter; // offset of next record to be reading
-    committedOffset = offset - 1; // offset of last record written to HDFS
+    committedConsumerOffset = committedConsumerOffset + recordCounter;
```

Comment:
If we keep `offset` and remove `committedConsumerOffset`, then we could keep the old line 779 (with it's comment) and remove the old 780.